runelite-client/src/main/java/net/runelite/client/plugins/banktags/BankTagsPlugin.java

```diff
@@ -151,11 +158,35 @@ BankTagsConfig getConfig(ConfigManager configManager)
 		return configManager.getConfig(BankTagsConfig.class);
 	}
 
+	@Override
+	public void startUp()
+	{
+		cleanConfig();
+		keyManager.registerKeyListener(this);
+		mouseManager.registerMouseWheelListener(this);
+		clientThread.invokeLater(tabInterface::init);
+		spriteManager.addSpriteOverrides(TabSprites.values());
+		overlayManager.add(tabQuantityOverlay);
+	}
+
+	@Override
+	public void shutDown()
```

Comment:
Why did these move

Reply:
Automatic formatter moved it. I moved it to it's original location now.

Reply:
It doesn't seem to be in its original location... In fact, there still seem to be a number of changes where a line, like an import, (or several lines in this case) were moved.

Reply:
Even after the new commit? If so I'll double check. I formatted the files and it must have shifted everything around because I didn't manually move methods.

Reply:
Moved everything correctly this time. Thanks!